### DISCUSSION TITLE
Help icon not aligning

### DISCUSSION THREAD

**Discussion by kharabela** [tailwindlabs/tailwindcss#13014] [Help] [answered]
https://play.tailwindcss.com/bbSmcbJOAS
<label class="mb-2 flex text-xs font-bold text-gray-700" for="entity-type"> Entity Type  .
If I remove flex from the class, the hover popup width appears as expected, however the help icon alignment breaks.
I want help icon aligned as well as the width of hover popup intact

**Comment by wongjn** [comment, marked as answer]
Consider applying `flex-grow: 1` to the position parent so that the tooltip has more horizontal space to expand into: https://play.tailwindcss.com/6lsKAeWToO
